schema/db_migration.sql

```diff
@@ -0,0 +1,212 @@
+-- ids:
+--  id: bigserial
+--  idx: bytea # 16 bytes, indexed, used for lookup
+--  remainder: bytea # remainder of binary data
+--  type: smallint # 0=eoa 1=contract 2=txhash 3=blockhash
+-- TAMCHAU:
+--   "type" is postgres internal key, should use other name -> "id_type".
+--   Should ENUM instead of smallint for better performance (don't need to convert int to value later).
```

Comment:
Enum is nice, but not portable as smallint, this is trade-off we need to consider depend on situation/application.